paywall/src/__tests__/unlock.js/MainWindowHandler/showCheckoutIframe.test.ts

```diff
@@ -0,0 +1,71 @@
+import FakeWindow from '../../test-helpers/fakeWindowHelpers'
+import { PaywallConfig } from '../../../unlockTypes'
+import IframeHandler from '../../../unlock.js/IframeHandler'
+import MainWindowHandler from '../../../unlock.js/MainWindowHandler'
+
+declare const process: {
+  env: {
+    PAYWALL_URL: string
+    USER_IFRAME_URL: string
+  }
+}
+
+describe('MainWindowHandler - showCheckoutIframe', () => {
+  process.env.PAYWALL_URL = 'http://paywall'
```

Comment:
HA! same thing as on this other PR ;)

Reply:
I had fixed this, but forgot to push the commit :)